tests/test_fixtures.py

```diff
@@ -279,6 +280,102 @@ def portfolio_problem_analog():
 
     return program
 
+@pytest.fixture()
+def bard_mpMILP_adapted_degenerate():
```

Comment:
if possible could you put where you found this problem in the doc string, if you found it anyware?